traffic_ops/app/lib/API/DeliveryServiceRegexes.pm

```diff
@@ -222,6 +262,18 @@ sub delete {
 		return $self->forbidden();
 	}
 
+	my $ds = $self->db->resultset('Deliveryservice')->find( { id => $ds_id } );
+	if ( !defined($ds) ) {
+		#allow deletion if the ds is not valid
```

Comment:
If sue to some bug or inconsitency - the a regex exists but the parent DS does not, we cannot check tenancy.
However we want to allow the deletion of the regex - which for itself exists.